explorer/models/azureRegistryNodes.ts

```diff
@@ -199,26 +199,36 @@ export class AzureRepositoryNode extends NodeBase {
                     tags = JSON.parse(body).tags;
                 }
             });
-
+            
+            const pool = new asyncPool(8);
             for (let i = 0; i < tags.length; i++) {
-                
-                let manifest = JSON.parse(await request.get('https://' + element.repository + '/v2/' + element.label + '/manifests/latest', {
-                    auth: { bearer: accessTokenARC }
-                }));
-
-                node = new AzureImageNode(`${element.label}:${tags[i]}`, 'azureImageNode');
-                node.azureAccount = element.azureAccount;
-                node.password = element.password;
-                node.registry = element.registry;
-                node.serverUrl = element.repository;
-                node.subscription = element.subscription;
-                node.userName = element.userName;
-                node.created = moment(new Date(JSON.parse(manifest.history[0].v1Compatibility).created)).fromNow();
-                imageNodes.push(node);
-
+                pool.addTask(async ()=>{                    
+                    let data = await request.get('https://' + element.repository + '/v2/' + element.label + `/manifests/${tags[i]}`, {
+                        auth: { bearer: accessTokenARC }
+                    });
+
+                    //Acquires each image's manifest to acquire build time.
+                    let manifest = JSON.parse(data);
+                    node = new AzureImageNode(`${element.label}:${tags[i]}`, 'azureImageNode');
+                    node.azureAccount = element.azureAccount;
+                    node.password = element.password;
+                    node.registry = element.registry;
+                    node.serverUrl = element.repository;
+                    node.subscription = element.subscription;
+                    node.userName = element.userName;
+                    node.created = moment(new Date(JSON.parse(manifest.history[0].v1Compatibility).created)).fromNow();
+                    imageNodes.push(node);
+                }
+                );
             }
-
+            await pool.scheduleRun();
+        }
+        function sortfunction(a: AzureImageNode, b : AzureImageNode):number{
+            if(a.serverUrl < b.serverUrl) return -1;
+            else if(a.serverUrl === b.serverUrl)return 0;
+            else return 1;
         }
```

Comment:
You may consider to install some typescript prettify extension to format the code